addons/qodot/src/resources/game-definitions/fgd/qodot_fgd_model_point_class.gd

```diff
@@ -0,0 +1,105 @@
+@tool
+class_name QodotFGDModelPointClass
+extends QodotFGDPointClass
+
+## Optional - if empty, will use the game dir provided when exported.
+@export_global_dir var trenchbroom_game_dir = ""
+@export var model_export_dir := "trenchbroom/export"
+## Scale expression applied to model in Trenchbroom. See https://trenchbroom.github.io/manual/latest/#display-models-for-entities for more info.
+@export var scale_expression := ""
+@export var generate_bounding_box := true
+@export var apply_rotation_on_import := true
+@export var generate_gd_ignore_file := false
+func build_def_text() -> String:
+	_generate_model()
```

Comment:
Is there a reason you have `_generate_model()` which only calls `_set_model()` instead of just calling `_set_model()` directly in `build_def_text()`?

Reply:
Not really, `_set_model()` used to have more inside and I prefer to have public functions as short as possible. I just hadn't bothered to just remove the function call. Will do soon.